src/sentry/workflow_engine/endpoints/validators/metric_alert_detector.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework import serializers
+
+from sentry.snuba.snuba_query_validator import SnubaQueryValidator
+from sentry.workflow_engine.endpoints.validators.base import (
+    BaseGroupTypeDetectorValidator,
+    NumericComparisonConditionValidator,
+)
+from sentry.workflow_engine.models.data_condition import Condition
+from sentry.workflow_engine.types import DetectorPriorityLevel
+
+
+class MetricAlertComparisonConditionValidator(NumericComparisonConditionValidator):
+    supported_conditions = frozenset((Condition.GREATER, Condition.LESS))
+    supported_results = frozenset((DetectorPriorityLevel.HIGH, DetectorPriorityLevel.MEDIUM))
```

Comment:
🤔 I wonder if this should be a `DetectorDataConditionValidator` - that extends from `DataConditionValidator`

I think for now this is 👍 and we can take another look as we get more examples of how this will work with the workflow data conditions and other detectors.